README.md

```diff
@@ -1,2 +1,130 @@
-# brandNewTodoApp
-this project is about a website app for listing to do activity for daily routine.
+<a name="readme-top"></a>
+
+  <h3><b>To do App</b></h3>
+
+</div>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](https://github.com/ZabihullahNooriWardak/AwesomeBook/blob/feature/LICENSE)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [ To Do App] <a name="about-project"></a>
+
+ > This is web based app which is used for storing(listing) daily activity.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+   #### HTML, CSS & Javascript.
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[User-friendly Interface]**
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://zabihullahnooriwardak.github.io/brandNewTodoApp/dist)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+>
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+ node.js installed
+ webpack installed
+
+### Setup
+
+Clone this repository to your desired folder:
+And go to that folder where you want the project for example if you want to have it in my-folder you can go like this 
+  C:\Users\Dell>
+  cd my-folder
+  C:\Users\Dell\my-folder
+After you can clone using this command  
+  git clone https://github.com/ZabihullahNooriWardak/toDoList
+
+
+### Install
+
+Install this project with:
+
+this project does not have need further installition just 
+you should have IDE and browser node installed and webpack too.
+
+### Usage
+
+to run this project you need Google chrome or other browser 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> 
+
+👤 **Zabi**
+
+- GitHub: [@githubhandle](https://github.com/ZabihullahNooriWardak)
+- Twitter: [@twitterhandle](https://twitter.com/ZabiNoo31954752)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zabi-noori-aa59a924a/)
```

Comment:
- [ ] The LinkedIn link you provided is broken, when clicked it does not show your account on the platform. Please Kindly provide the appropriate link 👍